Enterprise-CMCS/macpro-mdct-mfp
# Review & Submit Status Without Alert Check

### Description


The status for review & submit is now up to date, but i had to remove the checking of Initiative against the alert banner conditions. That conditional check will be addressed in another branch


https://github.com/Enterprise-CMCS/macpro-mdct-mfp/assets/127151429/a1377540-35e3-4cdf-b9a0-7ee298cc7914



### Related ticket(s)


CMDCT-2866

---
### How to test

1) Sign into MFP, create a new WP
2) Enter and fill out form information
3) Review & Submit status should accurately reflect the page infos except initiative

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
